releasenotes/notes/botocore-internals-49885b5842d1ee80.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    botocore: Introduces environment variable ``DD_BOTOCORE_INSTRUMENT_INTERNALS`` that opts into
+    tracing certain internal functionality
```

```suggestion
    tracing certain internal functionality.
```
Seems like we usually like periods.